widgetastic_manageiq.py

```diff
@@ -1589,3 +1589,19 @@ class chart(TimelinesChart):  # NOQA
     @property
     def is_displayed(self):
         return self.title.text == 'Timelines'
+
```

Comment:
@mfalesni @izapolsk @otsuman @sshveta You have all done some widgets now, I just prototyped this generic form view to eliminate duplicate button definitions in the various forms in cloud.instances and that will be in almost all forms for the application.

You can see the use above in cloud.instance, I have nested a view that inherits this form, and then defined the unique form widgets.

Thoughts? Any other prototypes/solutions for this at the moment that I'm missing?